src/vm/dynamicmethod.cpp

```diff
@@ -1098,8 +1084,7 @@ BYTE* LCGMethodResolver::GetCodeInfo(unsigned *pCodeSize, unsigned *pStackSize,
         OBJECTREF resolver = ObjectFromHandle(m_managedResolver);
         VALIDATEOBJECTREF(resolver); // gc root must be up the stack
 
-        DWORD initLocals = 0, EHSize = 0;
-        unsigned short stackSize = 0;
```

Comment:
This is the bug - the unmanaged `stackSize` was `unsigned short` vs. the managed one was `int`.

The rest is changes is cleanup.